packages/ibm-products/src/components/DataSpreadsheet/utils/moveColumnIndicatorLine.js

```diff
@@ -30,20 +30,53 @@ export const moveColumnIndicatorLine = ({
   const selectionAreaOrigin = ref.current.querySelector(
     `[data-matcher-id="${matcherId}"]`
   );
+
+  const listContainer = ref.current.querySelector(
+    `.${blockClass}__list--container`
+  );
+
+  let scrollSpeed = 10; // Scrolling speed
```

Comment:
One more thing, since these variables are not re-assigned a new value can you change from `let` to `const`?

Reply:
Sure I can do that